### ISSUE TITLE
Not compiling with Symbian^3 SDK  (Qt 4.7.3)

### ISSUE THREAD

**Issue by cip** [cip/WikiOnBoard#65]
Two problems:
1. In zimlib fstream.cpp:
Multiple locations (one as example):

```
C:\Symbian\Carbide\workspace\zimlib\src\fstream.cpp:95: Fehler:no match for 'operator<<' in 'std::operator<< [with _Traits = std::char_traits<char>](((std::basic_ostream<char, std::char_traits<char> >&)((std::basic_ostream<char, std::char_traits<char> >*)((std::basic_ostream<char, std::char_traits<char> >*)std::operator<< [with _Traits = std::char_traits<char>](((std::basic_ostream<char, std::char_traits<char> >&)(& msg.std::basic_ostringstream<char, std::char_traits<char>, std::allocator<char> >::<anonymous>)), ((const char*)"error ")))->std::basic_ostream<_CharT, _Traits>::operator<< [with _CharT = char, _Traits = std::char_traits<char>]((* __errno())))), ((const char*)" seeking to end in file ")) << ((zim::streambuf::FileInfo*)this)->zim::streambuf::FileInfo::fname'
```

Not checked in detail, but possible to continue by just commenting out lines. (Anyway only message for exceptions)
1. When building wikionboard:

```
  c:/qtsdk/symbian/tools/gcce4/bin/../lib/gcc/arm-none-symbianelf/4.4.1/../../../../arm-none-symbianelf/bin/ld.exe: warning: C:/QtSDK/Symbian/SDKs/Symbian3Qt473/epoc32/release/armv5/urel/usrt3_1.lib(ucppinit.o) uses variable-size enums yet the output is to use 32-bit enums; use of enum values across objects may fail
   C:/QtSDK/Symbian/SDKs/Symbian3Qt473/epoc32/release/armv5/urel/QtScroller.lib(qtscroller.o): In function `__static_initialization_and_destruction_0':
   C:/Symbian/Carbide/workspace/kineticscroller/src/qtscroller.cpp:292: undefined reference to `__aeabi_atexit'
   C:/Symbian/Carbide/workspace/kineticscroller/src/qtscroller.cpp:293: undefined reference to `__aeabi_atexit'
   C:/QtSDK/Symbian/SDKs/Symbian3Qt473/epoc32/release/armv5/urel/QtScroller.lib(qtscroller.o): In function `global constructors keyed to operator<<(QDebug&, QtScrollerPrivate::ScrollSegment const&)':
   C:/Symbian/Carbide/workspace/kineticscroller/src/qtscroller.cpp:2040: undefined reference to `__dso_handle'
   collect2: ld returned 1 exit status
```

Not really critical as anyway better to use symbian^1 sdk (as works on both S^1 and S^3). However, S^3 compile seems to be much
faster which would be nice for development.

**Comment by cip** [comment]
1. fixed in commit c03181f97a

**Comment by cip** [comment]
Note: WikiOnBoard 2 will be based on Qt Quick Components 1.1. 

These are only supported with
"Qt 4.7.4 for Symbian Anna (Qt SDK) Release" target
(Note: despite the name also pre-anna S^3 is supported)

For older devices (that is S^1) wikionboard 1.x is to be used.
(Note: quick actually supported with s^1 as well, but Qt Quick Components not supported)
